README.md

```diff
@@ -28,31 +28,7 @@ docker-compose up -d
 You may then query the service:
 
 ```bash
-curl -H "Accept: application/rss+xml" localhost:8000
-```
-
-Kibana's dev tools console may be helpful when developing:
-
-```bash
-docker run --rm -it --network host -e ELASTICSEARCH_HOSTS='["http://localhost:9200"]' kibana:7.16.3
-```
-
-Then create queries such as the one below in <http://localhost:5601/app/dev_tools#/console>.
-
-```json
-GET datasets/_search
-{
-  "query": {
-    "match_all": {}
-  },
-  "size": 100,
-  "sort": {
-    "harvest.firstHarvested": {
-      "order": "desc",
-      "unmapped_type": "long"
-    }
-  }
-}
+curl -H "Accept: application/rss+xml" localhost:8080
```

Comment:
Mangler atom mediatype

Reply:
Forsto ikke helt, kan du utdype?

Reply:
curl eksempel for atom?

Reply:
Ja, ok, bare et eksempel til? Har lagt det til.